flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -2199,12 +2199,15 @@ var A320_Neo_UpperECAM;
         getValue(_engine) {
             const imbalance = SimVar.GetSimVarValue("L:A32NX_ENGINE_IMBALANCE", "number");
             if (parseInt(imbalance.toString().substr(0, 1)) == _engine) {
-                var n2_i = parseFloat(imbalance.toString().substr(5, 2)) / 100;
+                let n2_i = parseFloat(imbalance.toString().substr(5, 2)) / 100;
             } else {
-                var n2_i = parseFloat(0);
+                let n2_i = parseFloat(0);
             }
             const name = "ENG N2 RPM:" + _engine;
-            const percent = SimVar.GetSimVarValue(name, "percent") - n2_i;
+            let percent = SimVar.GetSimVarValue(name, "percent") - n2_i;
```

Comment:
`n2_i` is always undefined at this point. Unlike `var` which is hoisted up, `let` is block-scoped.

Reply:
i stands for imbalance. Will make the changes. Thanks!

Reply:
Then I'd suggest renaming the variable to `n2Imbalance`.